src/ipc/ipc-zephyr.c

```diff
@@ -141,7 +132,6 @@ enum task_state ipc_platform_do_cmd(struct ipc *ipc)
 	}
 #endif
 
-	pm_policy_state_lock_put(PM_STATE_RUNTIME_IDLE, PM_ALL_SUBSTATES);
 	return SOF_TASK_STATE_COMPLETED;
```

Comment:
My understanding is that this would be a balanced call with get()/put() ? Can you describe the case where this was breaking ?

Reply:
It wasn't breaking. But when revisiting this code I realized that it can be done more easily. This way we remove the risk that put/get calls will not be balanced and it we generate less instructions.